id_minter/src/main/resources/db/migration/V2__different_ontologytype.sql

```diff
@@ -0,0 +1,26 @@
+-- This patch is preliminary work for supporting items in the ID minter.
+--
+-- Because a single MIRO record becomes both a work and an item, we:
+--
+--   * Add an ontologyType field so we can distinguish between an ID minted
+--     for a work and an item.  Everything in the ID minter database when
+--     we made this migration was a "Work", so we backfill existing rows,
+--     then set it to NULL for future rows.
+
+USE ${database};
+
+ALTER TABLE ${tableName}
+ADD COLUMN ontologyType varchar(255) DEFAULT 'Work' AFTER CanonicalID;
+
+ALTER TABLE ${tableName}
+ALTER COLUMN ontologyType SET DEFAULT null;
```

Comment:
Do we want this to be null by default? Do we expect to have things with an identifier and no type?